src/bacnet/basic/object/structured_view.c

```diff
@@ -462,7 +462,8 @@ bool Structured_View_Represents_Set(
  * @param  object_instance - object-instance number of the object
  * @return number of Subordinate_List elements
  */
-unsigned int Structured_View_Subordinate_List_Count(uint32_t object_instance)
+static unsigned int Structured_View_Subordinate_List_Count(
```

Comment:
Can these structured view functions (taking argument of object-instance) be added to header file instead?

Reply:
Definetly.